libexec/cli/image.create.exec

```diff
@@ -96,6 +102,19 @@ if [ -f "$SINGULARITY_IMAGE" ]; then
     fi
 fi
 
+# Get the inode option *before* creating the empty file
+# We want to exit before the 'dd' command
+INODE=""
+if [ -n "${SINGULARITY_INODESIZE:-}" ]; then
+    INODE="-i ${SINGULARITY_INODESIZE:-16384}"
```

Comment:
Is the `:-16384}"` part of this really necessary?  We just checked in the previous line to make sure that `SINGULARITY_INODESIZE` is not emtpy.

Reply:
No ... Call it me being psychotic an putting a default value in case it's empty even though I just checked that it wasn't empty...